tests/python_tests/test_chat_generate_api.py

```diff
@@ -37,34 +43,52 @@
 @pytest.mark.precommit
 @pytest.mark.nightly
 def test_chat_compare_with_HF(model_descr, generation_config: Dict):
-    device = 'CPU'
+    device = "CPU"
     chat_history_hf = []
     chat_history_ov = []
+<<<<<<< Updated upstream
     chat_prompt = ''
     
     # Will set add_special_tokens=False inside pipeline when start_chat() is called.
     model_id, path, tokenizer, model_opt, pipe = read_model((model_descr[0], model_descr[1] / '_test_chat'))
+=======
+    chat_prompt = ""
 
-    pipe.start_chat()    
+    # HF in chat scenario does not add special tokens, but openvino tokenizer by default is converted with add_special_tokens=True.
+    # Need to regenerate openvino_tokenizer/detokenizer.
+    model_id, path, tokenizer, model_opt, pipe = read_model(
+        (model_descr[0], model_descr[1] / "_test_chat"), add_special_tokens=False
+    )
+>>>>>>> Stashed changes
```

Comment:
That's why you shouldn't have extended the task with other code style checks. It becomes hard to manage.

Resolve the conflict.